app/boards/native_sim_64.overlay

```diff
@@ -0,0 +1,69 @@
+// SPDX-FileCopyrightText: 2025 Florian Limberger <flo@purplekraken.com>
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+#include <zephyr/dt-bindings/lvgl/lvgl.h>
+
+&flashcontroller0 {
+	reg = <0x00000000 DT_SIZE_K(1024 * 100)>;
+};
+
+&flash0 {
+	reg = <0x00000000 DT_SIZE_K(1024 * 100)>;
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		flashdisk_partition: partition@0 {
+			reg = <0x00000000 DT_SIZE_K(1024 * 100)>;
+		};
+	};
+};
+
+&sdl_dc {
+	compatible = "zephyr,sdl-dc";
+	height = <480>;
+	width = <800>;
+};
+
+/ {
+	keys: keys {
```

Comment:
May we could introduce a configuration option for the hardware buttons as the code is intended to work with a touchscreen as well. This would avoid the need to add the button definitions for the emulator